### ISSUE TITLE
Use of `class` as a meta property is limiting for other langauges

### ISSUE THREAD

**Issue by fosrias** [refractproject/refract-spec#37]
Not nice for ruby. Clobbers definition of `class` on an instance.

``` ruby
element.meta.class # => should be the class of the meta object
```

Ruby does not like #get and #set stylistically, and thus this is an issue.

**Comment by pksunkara** [comment]
What about `element.meta['class']`?

**Comment by danielgtaylor** [comment]
Actually, this fails even worse in some languages that have `class` as a reserved keyword. For example, in Python:

``` python
>>> class Foo:
...     def __init__(self):
...             self.class = 'abc'
  File "<stdin>", line 3
    self.class = 'abc'
             ^
SyntaxError: invalid syntax

>>> a = {}
>>> a.class
  File "<stdin>", line 1
    a.class
          ^
SyntaxError: invalid syntax
```

The equivalent there of `element.meta['class']` is pretty ugly:

``` python
value = getattr(element.meta, 'class')
```

The alternative is to expose a convenience property like `cls`. I think we should consider the naming because so many languages use the `class` keyword to define object types.

**Comment by kylef** [comment]
It would be nice to have type-safety instead of using dictionaries (in Swift at least), would be nice to find a new word for this.

**Comment by smizell** [comment]
We could do `classes` or `tags`

**Comment by fosrias** [comment]
@smizell I `classes` should work, but something nags me it is reserved somewhere as well. `tags` would be the least likely to collide with something. I concur with @kylef and @danielgtaylor that forcing the use of dictionaries is not a good design choice.

**Comment by smizell** [comment]
@fosrias Well, the core point of Refract was to get away from using dictionaries (specifically object literals in JS) and use actual class instances in OO languages for interacting with data. So I think these kinds of changes are important in making this possible.

Anyone want to write up a proposal for this? :)

**Comment by danielgtaylor** [comment]
I've opened an RFC for this change. ^

**Comment by kylef** [comment]
This can be closed, #40 has changed this to be classes.
